### PR TITLE
Introduced 2 new variables

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added two new variables:
### `tidb_opt_limit_push_down_threshold`

- Scope: SESSION | GLOBAL
- Default value: `100`
- Range: `[0, 2147483647]`
- This variable is used to push down the Limit and TopN operators to the threshold in TiKV.
- If the value of Limit or TopN is less than or equal to this threshold, then the Limit and TopN operators are forcibly pushed down to TiKV. This variable provides a solution when Limit or TopN cannot be pushed down due to estimation errors. 

### `tidb_opt_enable_correlation_adjustment`

- Scope: SESSION | GLOBAL
- Default value:  `ON`
- This variable is used for the optimizer whether to enable the method that estimates the number of rows based on column order correlation

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6796
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -996,6 +996,20 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join, Projection, and UnionAll.
 - When the aggregate operation is slow in query, you can set the variable value to ON.
 
+### `tidb_opt_limit_push_down_threshold`
+
+- Scope: SESSION | GLOBAL
+- Default value: `100`
+- Range: `[0, 2147483647]`
+- This variable is used to push down the Limit and TopN operators to the threshold in TiKV.
```
Comment (qw4990) [inline system-variables.md:1004]: `This variation is used to set the threshold that determines whether push the Limit or TopN operators down to TiKV`

Comment (qw4990) [reply system-variables.md:1004]: Rest LGTM

File: system-variables.md [hunk 2]
```diff
@@ -996,6 +996,20 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join, Projection, and UnionAll.
 - When the aggregate operation is slow in query, you can set the variable value to ON.
 
+### `tidb_opt_limit_push_down_threshold`
```
Comment (TomShawn) [inline system-variables.md:999]:
```suggestion
### tidb_opt_limit_push_down_threshold
```

File: system-variables.md (hunk 1 above)
Comment (TomShawn) [reply system-variables.md:1004]:
```suggestion
- This variable is used to set the threshold that determines whether to push the Limit or TopN operator down to TiKV.
```

File: system-variables.md [hunk 3]
```diff
@@ -996,6 +996,20 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+- Scope: SESSION | GLOBAL
+- Default value: `100`
+- Range: `[0, 2147483647]`
+- This variable is used to push down the Limit and TopN operators to the threshold in TiKV.
+- If the value of Limit or TopN is less than or equal to this threshold, then the Limit and TopN operators are forcibly pushed down to TiKV. This variable provides a solution when Limit or TopN cannot be pushed down due to estimation errors.
```
Comment (TomShawn) [inline system-variables.md:1005]:
```suggestion
- If the value of the Limit or TopN operator is smaller than or equal to this threshold, these operators are forcibly pushed down to TiKV. This variable resolves the issue that the Limit or TopN operator cannot be pushed down to TiKV partly due to wrong estimation. 
```

File: system-variables.md [hunk 4]
```diff
@@ -996,6 +996,20 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 
+### `tidb_opt_limit_push_down_threshold`
+
+- Scope: SESSION | GLOBAL
+- Default value: `100`
+- Range: `[0, 2147483647]`
+- This variable is used to push down the Limit and TopN operators to the threshold in TiKV.
+- If the value of Limit or TopN is less than or equal to this threshold, then the Limit and TopN operators are forcibly pushed down to TiKV. This variable provides a solution when Limit or TopN cannot be pushed down due to estimation errors. 
+
+### `tidb_opt_enable_correlation_adjustment`
```
Comment (TomShawn) [inline system-variables.md:1007]:
```suggestion
### tidb_opt_enable_correlation_adjustment
```

File: system-variables.md [hunk 5]
```diff
@@ -996,6 +996,20 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+- Scope: SESSION | GLOBAL
+- Default value:  `ON`
+- This variable is used for the optimizer whether to enable the method that estimates the number of rows based on column order correlation
```
Comment (TomShawn) [inline system-variables.md:1011]:
```suggestion
- This variable is used to control whether the optimizer estimates the number of rows based on column order correlation
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by qw4990 — APPROVED** [review]

**Pull request merged**